opencog/scm/opencog/base/utilities.scm

```diff
@@ -77,7 +77,7 @@
 (define-public (etv pos-count total-count) (cog-new-etv pos-count total-count))
 
 ; Fetch the mean, confidence and count of a TV.
-(define-public (tv-mean TV)
+(define-public (cog-tv-mean TV)
 "
```

Comment:
ahem, cough cough cough.   Did you actually look at what you are doing? 

This defines a backwards-compatibility function. You just changed it to call itself, in an infinite-recursive way.